src/web/modules/gateway_accounts/switch_psp/switch_psp.http.ts

```diff
@@ -6,6 +6,19 @@ import logger from "../../../../lib/logger";
 
 import stripeTestAccount from '../../stripe/test-account.http'
 
+async function setConnectorStripeSetup(gatewayAccountId: string, isTest: boolean) {
+  const defaultValue = isTest
```

Comment:
Any reason we have to create a const instead of using `isTest`?

Reply:
mainly to do with trying to better communicate the intent of the function, `isTest` is specific to the gateway account but also happens to be the boolean value that is used in the `updateStripeSetup` params. 

I was in two minds about it so can remove

Reply:
This method doesn't need to live/test gateway accounts, as the calling method will pass true/false based on the gateway account type. So `isTest` can be specific to the PATCH op.